actions/src/main/java/com/redhat/service/bridge/actions/webhook/WebhookActionValidator.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.service.bridge.actions.webhook;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.bridge.actions.ActionParameterValidator;
+import com.redhat.service.bridge.actions.ValidationResult;
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+
+@ApplicationScoped
+public class WebhookActionValidator implements ActionParameterValidator {
+
+    public static final String MISSING_ENDPOINT_PARAM_MESSAGE = "Missing or empty \"endpoint\" parameter";
+    public static final String MALFORMED_ENDPOINT_PARAM_MESSAGE = "Malformed \"endpoint\" URL";
+    public static final String INVALID_PROTOCOL_MESSAGE = "The \"endpoint\" protocol must be either \"http\" or \"https\"";
+
+    private static final Logger LOG = LoggerFactory.getLogger(WebhookActionValidator.class);
+    private static final String PROTOCOL_HTTP = "http";
+    private static final String PROTOCOL_HTTPS = "https";
+
+    @Override
+    public ValidationResult isValid(BaseAction baseAction) {
+        if (baseAction.getParameters() == null) {
+            return ValidationResult.invalid();
+        }
+
+        String endpoint = baseAction.getParameters().get(WebhookAction.ENDPOINT_PARAM);
+        if (endpoint == null || endpoint.isEmpty()) {
+            return ValidationResult.invalid(MISSING_ENDPOINT_PARAM_MESSAGE);
+        }
+
+        URL endpointUrl;
+        try {
+            endpointUrl = new URL(endpoint);
+        } catch (MalformedURLException e) {
+            String errorMsg = malformedUrlMessage(endpoint, e);
+            LOG.warn(errorMsg);
```

Comment:
nitpick: I think this is not a real `warn` message. It's a user's mistake a not a warning on our side. I'd probably not even log it

Reply:
You're right...I removed it :+1: